content/product-engineering/engineering/career-development/framework.md

```diff
@@ -343,7 +343,10 @@ To learn more, see ["Considerations for promotion" in our talent review process]
     <!-- IC4 -->
     <tr>
       <th id="ic4" scope="row" rowspan="3" class="level"><a class="anchor" href="#ic4"></a><abbr title="Individual Contributor">IC</abbr>4</th>
-      <td colspan="3" class="level-summary">A particularly experienced, impactful contributor.</td>
+      <td colspan="3" class="level-summary">
+        A particularly experienced, impactful contributor.<br>
+        IC4 is a terminal level: the team can have an unbounded number of IC4 engineers.
```

Comment:
I am not sure people will know what "terminal level" means. Can we not use that phrase and instead inline a definition of what we mean?

Reply:
The proposed change here actually makes it sound different than the definition I expected. To me, it means: a level at which we don't expect every engineer to advance beyond. That seems orthogonal to the question of whether the number of engineers at that level is bounded.

I also have trouble seeing how there is not an element of scarcity to IC4 as well, at least given the definitions that we have. E.g., "High-quality technical decision making, leading team-sized tasks that affect one or more complex systems or mission-critical areas" — is it really possible to have 6 people on one team with this kind of impact?

Reply:
> is it really possible to have 6 people on one team with this kind of impact?
Probably not, but also I don't think it's a good idea to have six IC4s on one team. That feels like a needless concentration of expertise that will leave those engineers under-utilized. (Unless the team has 20 people on it, I guess.)

What about something like this: "IC4 is the highest level that an engineer can get to based solely on their impact. Beyond IC4, both impact and business need determine the availability of those positions."

(It's not perfect, but I think it's a bit closer?)

Reply:
@chrispine Sounds good to me!

Reply:
yeah that makes sense

Reply:
```suggestion
        IC4 is the highest level that an engineer can get to based solely on their impact. Beyond IC4, both impact and business need determine the availability of those positions. 
```